apps/explorer/components/Block/index.tsx

```diff
@@ -99,6 +100,11 @@ export function Block({ block, blockID, currentHeight }: Props) {
                   transactions
                 </Badge>
               </Tooltip>
+              <Tooltip content={'hardfork version'}>
+                <Badge variant="simple" data-testid="explorer-block-version">
+                  {block.height >= allowHeight ? 'v2' : 'v1'}
```

Comment:
Should use `block.v2 === null ? 'v1' : 'v2'` instead. v1 blocks are allowed after the allow height.